test-case/check-reboot.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+##
+## Case Name: test reboot
+## Preconditions:
+##    N/A
+## Description:
+##    run reboot for the test
+## Case step:
+##    1. check system status is correct
+##    2. wait for the delay time
+##    3. trigger for the reboot
+## Expect result:
+##    Test execute without report error in the LOG
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=3
+
+OPT_OPT_lst['d']='delay'    OPT_DESC_lst['d']='delay after system bootup'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=2
```

Comment:
in some case delaying 2 second is not enough, I think setting at least 5 seconds by default is better.

Reply:
this delay is after `verify_lst`, so may be I move the delay before `verify_lst` is better also to load `check-runtime-pm-status` to check the PM status

Reply:
@Bin-QA @fredoh9 On some device the SOF load will take up to 30s.
As sof is always work on module mode, it will be probed very late. We also need to wait for the tplg to load. So I will suggest to set a timeout and give a time about how long SOF fully loaded took